src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java

```diff
@@ -392,9 +392,13 @@ private void handleSignToken(RoutingContext ctx) {
         permissionsSource.clearCacheUser(userId, tenant);
       }
       String username = payload.getString("sub");
-      String token = new AccessToken(tenant, username, userId).encodeAsJWT(tokenCreator);
-
-      JsonObject responseObject = new JsonObject().put("token", token);
+      Token token;
+      if (payload.getBoolean("dummy", Boolean.FALSE)) {
```

Comment:
Shouldn't this be `Boolean.TRUE`? Think you can also test for `type` to be `dummy`. Just to make things extra fun there are two types of `DummyToken`.

Reply:
mod-login does not supply dummy. The default ought to be false (not dummy). mod-users-bl OTOH provides dummy=true. In any case, it's a huge security risk to allow a dummy token to be created like that.

Reply:
Indeed you are right. Thanks for the links. I can see that password reset is sending a dummy token and gets an access token in return. Agree that using the type claim here would be wrong. Maybe adding a comment here would help a bit to explain what is going on.